spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'order_manager'
+
+describe 'User Stories' do
+
+  let(:text_double) { double :text_double, send: true }
+
+  context 'Allows a customer to see a list of dishes with prices' do
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+    it 'Allows method to be called without error' do
+      order = OrderManager.new
+      expect { order.see_menu }.not_to raise_error
+    end
+
+    it 'Pretty prints the current menu' do
+      order = OrderManager.new
+      expect(order.see_menu).to eq(
+        "1. Sea salt & caramel, £3.5\n"\
+        "2. Chilli & chocolate, £3.5\n"\
+        "3. Avocado cream, £4.0\n"\
+        "4. Coconut, £3.0\n"\
+        "5. Praline, £3.7\n"\
+        "6. Watermelon sorbet, £3.5\n"\
+        "7. Wild strawberry, £4.5\n"\
+        "8. Dark chocolate & cherry, £5.5"
+      )
+    end
+  end
+
+  context 'Allows a customer to place an order' do
+  # As a customer
+  # So that I can order the meal I want
+  # I would like to be able to select some number of several available dishes
+    context 'Allows customer to check the order total' do
+    # As a customer
+    # So that I can verify that my order is correct
+    # I would like to check that the total I have been given matches the sum of the various dishes in my order
```

Comment:
No need to include the user story comments in the test - the context/describe should detail the feature being tested.